egret/model_library/unit_commitment/reserve_requirement.py

```diff
@@ -16,11 +16,10 @@
 from .reserve_vars import check_reserve_requirement
 component_name = 'reserve_requirement'
 
-#TODO: this doesn't check if storage_services is added first, 
-#      but this will only happen when there are storage_services!
 @add_model_attr(component_name, requires = {'data_loader': None,
                                             'reserve_vars': None,
-                                            'non_dispatchable_vars': None
+                                            'non_dispatchable_vars': None,
```

Comment:
Comment to objective.py above applies here - seems like the changes to reserve_requirement.py aren't part of fuel supply modeling.